widget/embedded/src/components/BlockchainsSection/BlockchainsSection.styles.ts

```diff
@@ -3,5 +3,5 @@ import { styled } from '@rango-dev/ui';
 export const Blockchains = styled('div', {
   display: 'grid',
   gap: '$10',
-  gridTemplateColumns: 'repeat(6, minmax(0, 1fr))',
+  gridTemplateColumns: 'repeat(6, 1fr)',
```

Comment:
Considering the comments made earlier, could you please reverse these alterations?

Reply:
fixed